Homebrew/Homebrew.munki.recipe.yaml

```diff
@@ -28,28 +28,40 @@ Process:
         installcheck_script: |
           #!/bin/zsh
 
+          autoload is-at-least
+
           installed=1
           not_installed=0
           target_version="%version%"
 
+          # Get the current user
+          current_user="$(/usr/bin/stat -f%Su /dev/console)"
+
+          # If we're at the login window, Homebrew's .pkg will fail to install
+          if [[ $current_user == "root" || $current_user == "_mbsetupuser" || $current_user == "loginwindow" ]]; then
+            /bin/echo "No user account is logged in, so considering \"installed\" for now."
+            exit "$installed"
+          fi
+
           if [[ $(uname -m) == "x86_64" ]]; then
             brew="/usr/local/Homebrew/bin/brew"
           else
             brew="/opt/homebrew/bin/brew"
           fi
 
           if [[ ! -f "$brew" ]]; then
+            /bin/echo "$brew doesn't exist."
             exit "$not_installed"
           fi
 
-          current_version=$($brew --version | /usr/bin/head -n 1 | /usr/bin/awk '{ print $2 }')
+          current_version=$(/usr/bin/su -l $current_user -c "$brew --version" | /usr/bin/head -n 1 | /usr/bin/awk '{ print $2 }')
```

Comment:
I don't know if this is better or worse or neither compared to `sudo -u $current_user $brew --version` but it's fine.

Reply:
I'm open to whatever runs the command as the logged-in user, but the original (pre-PR) version runs the command as root, not as user.